resteasy-jaxrs/src/main/java/org/jboss/resteasy/plugins/delegates/ClientCookie.java

```diff
@@ -0,0 +1,22 @@
+package org.jboss.resteasy.plugins.delegates;
+
+import javax.ws.rs.core.Cookie;
+import javax.ws.rs.core.NewCookie;
+
+/**
+ * 
+ * @author <a href="mailto:ron.sigal@jboss.com">Ron Sigal</a>
+ * @date May 20, 2016
+ */
+public class ClientCookie extends NewCookie
```

Comment:
You're quite right. What threw me was that org.jboss.resteasy.plugins.delegates.CookieHeaderDelegate.toString() was using ServerCookie.appendCookieValue(), so I created a ClientCookie with its own delegate. I should have just fixed CookieHeaderDelegate. Stay tuned.